resources/js/Hooks/useMetaMask.ts

```diff
@@ -62,7 +62,8 @@ const networkConfigs: Record<number, object> = {
     },
 };
 
-const hasMetaMask = (): boolean => window.ethereum !== undefined;
+// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-explicit-any
```

Comment:
Note: using explicit `any` locally is not necessary since it properly handles the ethereum type that is defined on the `resources/types/globals.d.ts` but for some reason does not work here and creates a linting issue